### ISSUE TITLE
Reproduction of pattern prediction results

### ISSUE THREAD

**Issue by yichenwang231** [LirongWu/PSC-CPI#5]
Very nice work,
 I'm using 
`python train.py --task_mode 1 --modality seq_str_linear --pre_train 1 --seq_encoder HRNN --str_encoder GAT`
 to reproduce the results of the pattern prediction in the fine-tuning phase of 60 epochs, the loss suddenly becomes very large and the average AUPRC is only about 15, in your paper it is 18, is there a problem with the network and hyperparameters I am using?

**Comment by LirongWu** [comment]
Apologize for the delayed response. Reported performance comes from more training epochs (hyperparameter defaults to 200), and fine-tuning 60 epochs may not be quite enough. I have tested on a local platform but not seen a sudden increase in loss. Considering that different platform devices affect a lot, perhaps trying a different random seed, or finding a better combination of hyperparameters in the search space as described in the paper could help resolve this issue. I hope this helps.
